pycobertura/filesystem.py

```diff
@@ -68,3 +69,52 @@ def open(self, filename):
             f.close()
         except KeyError:
             raise self.FileNotFound(filename)
+
+
+def get_root_path(repository_folder):
+    command = "git rev-parse --show-toplevel"
+    return get_output(command, working_folder=repository_folder).rstrip()
+
+
+class GitFileSystem(FileSystem):
+    def __init__(self, repo_folder, commit_id=None):
```

Comment:
Should we make `commit_id` required? It seems that we're using it unconditionally.

```suggestion
    def __init__(self, repo_folder, commit_id):
```

Reply:
👍